### PR TITLE
Feat/accessibility ids

### PR DESCRIPTION
**Pull request by sayaansaha**
update accessibility Ids of buttons to match that of android test app.

### CODE REVIEW

**Comment by e7mac** [comment]
Did you mean to make changes to `Leanplum-iOS-SDK-Example/Configure.h` ?

**Review by alexisoyama — APPROVED** [review]
Again, LGTM as long as it works on STATIC. Maybe Xcode fixed it (?)

File: Leanplum-iOS-SDK-Example/Configure.h [hunk 1]
```diff
@@ -16,7 +16,7 @@
 #define LP_PRODUCTION_KEY @"prod_A1c7DfHO6XTo2BRwzhkkXKFJ6oaPtoMnRA9xpPSlx74"
 #define LP_DEVELOPMENT_KEY @"dev_Bx8i3Bbz1OJBTBAu63NIifr3UwWqUBU5OhHtywo58RY"
 
-#define LP_API_HOST_NAME @"127.0.0.1:3000"
+#define LP_API_HOST_NAME @"10.0.2.2:3000"
```
Comment (Enuviel) [inline Leanplum-iOS-SDK-Example/Configure.h]: why?

Comment (sayaansaha) [reply Leanplum-iOS-SDK-Example/Configure.h]: To be the same as Android Example app, and It works with api server when set to 10.0.2.2:3000.

Comment (Enuviel) [reply Leanplum-iOS-SDK-Example/Configure.h]: why does it need to be same?  why we can't use 127.0.0.1 for iOS?

Comment (sayaansaha) [reply Leanplum-iOS-SDK-Example/Configure.h]: Doesnt connect to API server, connects with 10.0.2.2:3000, hence why ben changed it for android i believe. Im going to merge my changes.

Comment (Enuviel) [reply Leanplum-iOS-SDK-Example/Configure.h]: how tests worked before?

Comment (Enuviel) [reply Leanplum-iOS-SDK-Example/Configure.h]: we do not need this changes. For iOS localhost should work, for Android it special situation.

Comment (Enuviel) [reply Leanplum-iOS-SDK-Example/Configure.h]: @e7mac please correct me if i am wrong and you cant connect from ios simulator to "127.0.0.1:3000"

Comment (sayaansaha) [reply Leanplum-iOS-SDK-Example/Configure.h]: @Enuviel feel free to test it yourself, I have and I discovered that 127.0.0.1:3000 does not work and that 10.0.2.2:3000 does work.

Comment (sayaansaha) [reply Leanplum-iOS-SDK-Example/Configure.h]: I reverted it so that we can move on

**Pull request merged**